src/php/views/page.twig

```diff
@@ -3,12 +3,18 @@
 {% block content %}
 	<div>
 		{% if post.thumbnail %}
-			<img
-				src="{{ post.thumbnail.src|resize(960) }}"
-				{% if post.thumbnail.alt and post.thumbnail.alt|length > 0 %}
-					alt="{{ post.thumbnail.alt }}"
-				{% endif %}
-			/>
+		<img
+			src="{{ post.thumbnail.src('large') }}"
+			srcset="{{ post.thumbnail.src('thumbnail') }} 150w,
+					{{ post.thumbnail.src('medium') }} 300w,
+					{{ post.thumbnail.src('medium_large')}} 768w,
```

Comment:
This should be `{{ post.thumbnail.src('medium_large') }} 768w,` to make the linter happy but I can no longer push.